openapi/types.go

```diff
@@ -12,7 +12,7 @@ func TypeString(typ string, format string) string {
 		return "float64"
 	case "string":
 		if format == "date-time" {
-			return "time.Time"
+			return "*time.Time"
```

Comment:
I guess it would be nice not to use `*time.Time` when a field is required, but I'm guessing that would be too much hassle.